wrecker.go

```diff
@@ -61,21 +62,33 @@ func (w *Wrecker) Delete(endpoint string) *Request {
 }
 
 func (w *Wrecker) sendRequest(r *Request) (*http.Response, error) {
-	var contentType string
+
+	var contentType string = "application/x-www-form-urlencoded"
 	var bodyReader io.Reader
 	var err error
 
-	// Empty Body means that we're posting Params via Form encoding
-	if r.HttpBody == nil {
-		bodyReader = strings.NewReader(r.Params.Encode())
-		contentType = w.DefaultContentType
-	} else {
-		// Otherwise, we're sending a request body
-		contentType = "application/json"
-		bodyReader, err = prepareRequestBody(r.HttpBody)
-
-		if err != nil {
-			return nil, err
+	// GET methods don't have an HTTP Body.  For all other methods,
+	// it's time to defined the body content.
+	if r.HttpVerb != GET {
```

Comment:
I could imagine refactoring this at a later date into a function, but for now this is good.